server.js

```diff
@@ -68,4 +68,5 @@ app.use(app.router);
 http.createServer(app).listen(port, function(){
   console.log("Lens running on port " + port);
   console.log("http://127.0.0.1:"+port+"/");
+  console.log("To open a local example file goto http://127.0.0.1:"+port+"?url=data/ams/bproc10/bproc10.xml");
```

Comment:
Fine if we just revert them or you want them to disappear from the commit history entirely? Would need to rewrite the devel branch to do that.